safe_transaction_service/history/views_v2.py

```diff
@@ -50,7 +53,7 @@ def get(self, request, address):
             self.request.query_params.get("exclude_spam", False)
         )
 
-        paginator = pagination.ListPagination(self.request)
+        paginator = pagination.ListPagination(self.request, max_limit=10)
```

Comment:
Should we keep the same criteria for max_limit in all endpoints? In any case, we should document it in the swagger. WDYT?

Reply:
I don´t think that we should have max_limit 10 for all the endpoints, we are setting just 10 for collectibles endpoint because have several outgoing requests to third parties endpoints to fill the metadata information of each collectible.